go/api/base_client.go

```diff
@@ -7983,3 +7983,42 @@ func (client *baseClient) ScriptExists(
 
 	return handleBoolArrayResponse(response)
 }
+
+// ScriptFlush removes all the scripts from the script cache.
```

Comment:
```suggestion
// Removes all the scripts from the script cache.
```
We usually don't copy API name to docs

Reply:
If you will change it - update others too

Reply:
Updated.